### DISCUSSION TITLE
Dark navbar but light dropdowns?

### DISCUSSION THREAD

**Discussion by mholt** [tabler/tabler#1601] [Q&A]
Is it possible to restore the pre-beta-18 look of having a dark navbar but a light dropdown?

(Sorry I don't have a screenshot handy, switching back a version would be a bit tedious for me at the moment -- case of the Mondays I guess)

But hopefully the demo page can illustrate: https://preview.tabler.io/layout-navbar-overlap.html?theme=light

The dark navbar is useful for contrast, but on a light theme I want light dropdowns coming from that navbar. (Click on the profile info or notification bell for example)

Is it possible to restore that look (maybe with a CSS class or HTML attribute or something)?

**Comment by rjd22** [comment]
It should be possible but not out of the box. You will have to make your own css to also account for the dark mode and how you want to display the menu in that case.

**Comment by mholt** [comment]
I mean, is the change intentional? I'm asking for Tabler to restore the old behavior, rather than me having to write my own CSS for this (which mostly defeats the purpose of using a CSS framework).

Basically, it should be possible to change the color of the navbar without changing its menus too.

**Comment by rjd22** [comment]
Hmm that is a good question. I would say it's intentional but does mean it's breaking backwards compatibility.

@codecalm Can you chime in? So on a light theme the dropdowns of the dark navbar are also dark. Is that intentional?

**Comment by codecalm** [comment]
it's bug, dropdown should be white, I'll fix it

**Reply by mholt** [reply]
Yay :smiley: Thank you!!

**Comment by codecalm** [comment]
fixed in https://github.com/tabler/tabler/pull/1604

**Comment by mholt** [comment]
@codecalm I'm probably missing something silly, but: I don't think it is fixed yet?

Is this preview supposed to have the fix? https://tabler-git-dev-dropdown-light-fix-tabler-io.vercel.app/layout-navbar-overlap.html

Notice that the theme is light and the dropdown is dark. I would expect the dropdown to be light (like it was in a previous beta version).

**Comment by mholt** [comment]
@codecalm This issue is still present here: https://preview.tabler.io/layout-navbar-overlap.html?theme=light

The theme is light, yet the dropdowns are dark.
